components/collapsible/collapsible.test.js

```diff
@@ -67,45 +70,72 @@ describe('Dialtone vue Collapsible Component Tests', function () {
 
   afterEach(async function () {
     propsData = baseProps;
+    slots = { contentOnExpanded, contentOnCollapsed };
     _clearChildWrappers();
   });
 
   describe('Test default rendering', function () {
     it('should render the component', function () {
-      assert.exists(wrapper, 'wrapper exists');
+      assert.isTrue(wrapper.exists(), 'wrapper exists');
     });
 
     it('should render the anchor', function () {
-      assert.exists(anchorElement, 'anchor exists');
+      assert.isTrue(anchorElement.exists(), 'anchor exists');
+    });
+
+    it('should render the content on expanded element', function () {
+      assert.isTrue(contentOnExpandedElement.exists(), 'content exists');
     });
 
-    it('should render the content', function () {
-      assert.exists(contentElement, 'content exists');
+    it('should NOT render the content on collapsed element', function () {
+      assert.isFalse(contentOnCollapsedElement.exists(), 'collapsed content does not exist');
     });
   });
 
-  describe('When scoped slot is provided', function () {
+  describe('When scoped slot is provided for anchor', function () {
     beforeEach(function () {
-      const anchor = '<button data-qa="anchor-element">click me</button>';
-      scopedSlots = { anchor };
+      slots = { anchor };
+      _mountWrapper();
     });
 
-    it('should render the scoped slot', function () {
-      assert.exists(anchorSlotElement, 'anchor slot exists');
+    it('should render the anchor slot', function () {
+      assert.isTrue(anchorSlotElement.exists(), 'anchor slot exists');
+    });
+  });
+
+  describe('When scoped slot is not provided for any content', function () {
+    beforeEach(function () {
+      slots = { };
+      _mountWrapper();
+    });
+
+    it('should NOT render any content if no slot is passed', function () {
+      contentOnCollapsedElement = wrapper.find('[data-qa="content-on-collapsed-element"]');
+      contentOnExpandedElement = wrapper.find('[data-qa="content-on-expanded-element"]');
+      assert.isFalse(contentOnExpandedElement.exists());
+      assert.isFalse(contentOnCollapsedElement.exists());
     });
   });
 
   describe('Test open prop undefined', function () {
```

Comment:
I'm actually setting the default (in `afterEach`) to set both `contentOnCollapsed` and `contentOnExpanded`, that's why I had to create a separate test where both are undefined.

Reply:
Yeah I'm a bit confused why we are doing this in `afterEach` instead of `beforeEach`. Generally we set state like this in `beforeEach` before the `mountWrapper();`

Reply:
Agreed, I don't think we need the `afterEach` here. I removed it and updated the `mountWrapper` to contain prop/slot defaults.